src/dynamic_sharing_service/attribute.definition.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const Validation = require('../yoti_common/validation');
+
+class AttributeDefinition {
+  constructor(name) {
+    Validation.isString(name, 'name');
+
+    this.name = name;
+  }
+
+  getName() {
+    return this.name;
+  }
+
+  toJSON() {
+    return {
+      name: this.name,
+    };
+  }
+}
+
+module.exports = AttributeDefinition;
```

Comment:
Should this file be in `src/data_type/` if it's to be reused for #110 ?

Reply:
Fixed in https://github.com/getyoti/yoti-node-sdk/pull/110/commits/8b8c3f343791dacba873ac9d5eab36cd76575698